src/Nokia.xml

```diff
@@ -0,0 +1,470 @@
+        </li>
+        <list>
+          <li>
+            <b>a)</b>
+            <p>which is directly or indirectly controlling such party;</p>
+          </li>
+          <li>
+            <b>b)</b>
+            <p>which is under the same direct or indirect ownership or control as such party; or</p>
+          </li>
+          <li>
+            <b>c)</b>
+            <p>which is directly or indirectly owned or controlled by such party.</p>
+            <p>For these purposes, an entity shall be treated as being controlled by another if that other
+               entity has fifty percent (50%) or more of the votes in such entity, is able to direct its
+               affairs and/or to control the composition of its board of directors or equivalent
+               body.</p>
+            <p>&quot;Commercial Use&quot; shall mean distribution or otherwise making the Covered Software
+               available to a third party.</p>
+            <p>&quot;Contributor&quot; shall mean each entity that creates or contributes to the creation of
+               Modifications.</p>
+            <p>&quot;Contributor Version&quot; shall mean in case of any Contributor the combination of the
+               Original Software, prior Modifications used by a Contributor, and the Modifications made
+               by that particular Contributor and in case of Nokia in addition the Original Software in
+               any form, including the form as Exceutable.</p>
+            <p>&quot;Covered Software&quot; shall mean the Original Software or Modifications or the
+               combination of the Original Software and Modifications, in each case including portions
+               thereof.</p>
+            <p>&quot;Electronic Distribution Mechanism&quot; shall mean a mechanism generally accepted in
+               the software development community for the electronic transfer of data.</p>
+            <p>&quot;Executable&quot; shall mean Covered Software in any form other than Source Code.</p>
+            <p>&quot;Nokia&quot; shall mean Nokia Corporation and its Affiliates.</p>
+            <p>&quot;Larger Work&quot; shall mean a work, which combines Covered Software or portions
+               thereof with code not governed by the terms of this License.</p>
+            <p>&quot;License&quot; shall mean this document.</p>
+            <p>&quot;Licensable&quot; shall mean having the right to grant, to the maximum extent possible,
+               whether at the time of the initial grant or subsequently acquired, any and all of the
+               rights conveyed herein.</p>
+            <p>&quot;Modifications&quot; shall mean any addition to or deletion from the substance or
+               structure of either the Original Software or any previous Modifications. When Covered
+               Software is released as a series of files, a Modification is:</p>
+          </li>
+          <li>
+            <b>a)</b>
```

Comment:
This should be within a new list, not part of the existing one, because otherwise we have a), b), c), a).